labs/optimizely-data-cli/README.md

```diff
@@ -0,0 +1,162 @@
+
+- [jq](https://stedolan.github.io/jq/)
+- [curl](https://curl.haxx.se/)  (Note: All versions of OS X starting with Jaguar come with curl installed)
+- the [Amazon AWS CLI](https://aws.amazon.com/cli/) (v2+) 
+
+## Downloading `oevents`
+
+You can download a zipped directory containing the `oevents` CLI [here](https://library.optimizely.com/labs/artifacts/optimizely-data-cli/optimizely-data-cli.zip).
+
+`oevents` lives in the  [Optimizely Labs Github repository](https://github.com/optimizely/labs/tree/master/labs/optimizely-data-cli).  You can also access it by cloning the repository or downloading the `oevents` script directly from the Github web interface.
+
+## Installation 
+
+`oevents` is a bash script. To run it, you'll need to make sure it's executable.  Assuming `oevents` is in your working directory, you can do this with the following command:
+
+```sh
+$ chmod +x oevents
+```
+
+To run `oevents`, you can specify the script's path explicitly like this (again, assuming `oevents` is in your working directory):
+
+```sh
+$ ./oevents help
+```
+
+Or you can add it to a directory in your `PATH` environment variable.  
+
+```sh
+$ sudo cp oevents /usr/local/bin/
+$ oevents help
+```
+
+## Get Help
+
+For usage instructions, use the `help` command:
+
+```sh
+$ oevents help
+```
+
+## Authenticating
+
+Optimizely Enriched Event data is served via [Amazon S3](https://aws.amazon.com/s3/). `oevents` uses the [Amazon AWS CLI](https://aws.amazon.com/cli/) to interact with S3, which requires authentication.  You can authenticate to AWS in three ways:
+
+1. (Recommended) Pass your [Optimizely Personal Access Token](https://docs.developers.optimizely.com/web/docs/personal-token) to `oevents` via the `OPTIMIZELY_API_TOKEN` environment variable:
+
+    ```sh
+    $ export OPTIMIZELY_API_TOKEN=<token>
+    $ oevents ls
+    ```
+2. Pass your [Optimizely Personal Access Token](https://docs.developers.optimizely.com/web/docs/personal-token) and [Optimizely Account ID](https://help.optimizely.com/Account_Settings/Account_name%2C_ID%2C_and_multi-account_login_in_Optimizely_X) via the `--token` and `--account-id` command line arguments:
+
+    ```sh
+    $ oevents ls --token <token> --account-id 12345
+    ```
+3. Provide your AWS credentials directly. See the [AWS user guide](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html) for instructions.
+
+If you chose methods 1 or 2, `oevents` will acquire temporary AWS credentials using the [Optimizely Authentication API](https://docs.developers.optimizely.com/optimizely-data/docs/authentication-api).
+
+
+**Note:** If you have an Optimizely Personal Access Token, you can use the `oevents auth` command to acquire temporary AWS credentials that may be used directly with the AWS CLI or client libraries in your ETL pipeline:
```

Comment:
I'm not sure how this is different / separate from option 2 above.

Reply:
Good point--I rewrote the "note" in order to clarify.  Let me know if that's still confusing.